src/sync-n-parent.js

```diff
@@ -0,0 +1,27 @@
+AFRAME.registerComponent('sync-n-parent',
```

Comment:
To make sure I understand, I believe this component does:

- acquire userId
- wait for connection
- instantiate an entity on remote clients that has 
  - all of our components except for the sync components 
  - the userId on any n-skeleton-parents set to our userId

My expectation for sync-n-parent

- Synchronizes the properties of n-parent (side / part / index)